dashboard/src/app/dashboard/last-workspaces/last-workspaces.html

```diff
@@ -1,4 +1,13 @@
-<che-panel che-title="Recent Workspaces" che-title-icon="material-design icon-ic_widgets_24px">
+<div class="dashboard-last-workspaces-content">
```

Comment:
I think I have my question as I'm seing other changes using duplicates of the style so I think we need reusable component there